### ISSUE TITLE
Show the timezone for build time

### ISSUE THREAD

**Issue by tonglil** [drone/drone#2444]
It would be great if Drone can show the timezone of the builds/the system it is running on when users hover over the timestamps.

<img width="353" alt="image" src="https://user-images.githubusercontent.com/3250776/41735509-ba9a48ec-753e-11e8-83bc-c3646a16a53d.png">

In this case, I would like to see `2018-06-20 14:57 UTC`.

**Comment by bradrydzewski** [comment]
agreed, although instead of UTC we should be able to use the browser's timezone, since drone stores all times as unix timestamps
